src/include/daos_fs_sys.h

```diff
@@ -27,15 +25,15 @@ extern "C" {
 #include <daos.h>
 #include <daos_fs.h>
 
-#define DFS_SYS_NO_CACHE 1
-#define DFS_SYS_NO_LOCK 2
+/** Mount flags for dfs_sys_mount. By default, mount with caching and locking turned on. */
+#define DFS_SYS_NO_CACHE 1 /** Turn off directory caching */
+#define DFS_SYS_NO_LOCK 2  /** Turn off locking. Useful for single-threaded applications. */
```

Comment:
The annotation here is wrong, I think you need this
```suggestion
#define DFS_SYS_NO_LOCK 2  /**<Turn off locking. Useful for single-threaded applications. */
```

Reply:
Ah, Thanks!